src/components.js

```diff
@@ -125,23 +125,34 @@ export class Diagram extends Component {
     render() {
         let options = this.props.options == null ? '' : `[${this.props.options}]`
 
+        let cells = this.toArray().map(entries => entries.map(entry =>
+            entry == null ? ''
+            : [
+                (() => {
+                    let value = entry.node.props.value;
+                    let [w1, w2] = value != null
+                        && needWrapChars.some(c => value.includes(c))
+                        ? ['{', '}'] : ['', '']
+
+                    return value != null ? `${w1}${value}${w2}` : '{}'
+                })(),
+                ...entry.edges.map(e => renderEdge(e, this.props.co))
+            ].join(' ')
+        ))
+
+        if(this.props.align && cells.length > 0) {
+            for(let j in cells[1]) {
```

Comment:
A `for` `in` loop over arrays is considered [bad practice](https://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-a-bad-idea). Please use `arr.forEach()` or a normal `for` loop.